### PR TITLE
[Federal_Trade_Commission] Reactivate ruleset.

### PR DESCRIPTION
**Pull request by bardiharborow**
Remove dead hostnames and re-enable.

### CODE REVIEW

**Comment by J0WI** [comment]
Some tests are still failing

**Comment by bardiharborow** [comment]
`eis.ftc.gov` (some internal subdomain) was being problematic, so I've dropped it.

File: src/chrome/content/rules/Federal_Trade_Commission.xml [hunk 1]
```diff
@@ -43,14 +35,11 @@ Fetch error: http://new.ftc.gov/ => https://new.ftc.gov/: (6, 'Could not resolve
 	<target host="consumidor.ftc.gov" />
 	<target host="www.consumidor.ftc.gov" />
 	<target host="edit.ftc.gov" />
-	<target host="new.edit.ftc.gov" />
-	<target host="eis.ftc.gov" />
```
Comment (J0WI) [inline src/chrome/content/rules/Federal_Trade_Commission.xml]: This host serves an incomplete certificate chain. Please add a comment about this issue.

Comment (bardiharborow) [inline src/chrome/content/rules/Federal_Trade_Commission.xml]: Done.

**Pull request merged**